lib/screens/v2/widget_instance/cr_departures.ex

```diff
@@ -0,0 +1,150 @@
+defmodule Screens.V2.WidgetInstance.CRDepartures do
+  @moduledoc false
+
+  alias Screens.Config.V2.CRDepartures
+  alias Screens.V2.Departure
+  alias Screens.V2.WidgetInstance.CRDepartures, as: CRDeparturesWidget
```

Comment:
Could you move this alias inside of the `defimpl` block to avoid confusion?
(Also huh, I'm surprised that putting the alias out here makes it available inside the `defimpl` 🤔)

Reply:
This pattern was taken from WidgetInstance.Departures